File: sb_explorer/sb_explorer/EXObjects/Sounbanks/EXSound.cs
using System.Collections.Generic;

namespace sb_explorer.EXObjects
{
    public class EXSound
    {
        public short DuckerLenght;
        public short MinDelay;
        public short MaxDelay;
        public short InnerRadiusReal;
        public short OuterRadiusReal;
        public sbyte ReverbSend;
        public sbyte TrackingType;
        public sbyte MaxVoices;
        public sbyte Priority;
        public sbyte Ducker;
        public sbyte MasterVolume;
        public ushort Flags;
        public uint Hashcode;
        public uint FlagsOffset;
        public uint SamplePoolOffset;
        public uint BinaryLength;
        public byte[] BinaryData;

        public List<EXSample> Samples = new List<EXSample>();
    }
}

File: sb_editor/sb_editor/Classes/SoundbankExporter/EXSound.vb
Public Class EXSound
    Public HashCode As UInteger
    Public Ducker As SByte
    Public DuckerLength As Short
    Public Flags As UShort
    Public InnerRadius As Short
    Public MasterVolume As SByte
    Public MaxDelay As Short
    Public MaxVoices As SByte
    Public MinDelay As Short
    Public OuterRadius As Short
    Public Priority As SByte
    Public ReverbSend As SByte
    Public TrackingType As SByte
    Public HasSubSfx As Boolean
    Public Samples As New List(Of EXSample)()
End Class
